src/components/Filters/DateRangePicker/index.js

```diff
@@ -40,6 +39,23 @@ class DateRangePicker extends React.Component{
             </div>
         );
     }
-}
+};
+
+DateRangePicker.propTypes = {
+
+};
+
+const mapStateToProps = state => {
+    return {
```

Comment:
you don't need use keyword 'return' here, and can use construction () => ({
    a: '1',
    b: '2'
}) instead

Reply:
Done

Reply:
There is no pros of using shorter form. Otherwise, using of return and brackets makes construction more clear and allows collapsing in IDEs